mmtrack/apis/inference.py

```diff
@@ -31,6 +31,7 @@ def init_model(config, checkpoint=None, device='cuda:0', cfg_options=None):
         config.merge_from_dict(cfg_options)
     if 'detector' in config.model:
         config.model.detector.pretrained = None
+    config.model.pretrains = {}
```

Comment:
We cannot disable the `pretrains` because for MOT models, we are using it to load ckpts.

Reply:
Even during inference? My intention here was to avoid loading FlowNet in the DFF VID model as I assumed it's weights had been updated during the traing and was thus contained in the snapshot?